CHARTER.md

```diff
@@ -0,0 +1,71 @@
+  
+
+### 1. Scope
+
+The scope of Editing TF covers multiple aspects of editing, which may  include:
+
+- Textual input and text manipulation 
+- Text editing related events 
+- Selection 
+- Clipboard 
+- Spellcheck and grammar checking 
+- Reusing or discarding parts of or entire pre-existing editing APIs to serve as primitives of editing systems 
+- Standardization of under-specified existing editing related features 
+- Highlighting parts of text ranges without inducing DOM mutations. 
+
+Experts who are contributing members of the task force will contribute to documents produced by the group, as described in the section on deliverables below. 
+ 
+
+### 2. Deliverables
+
+This Task Force will
+- ensure that browsers are developing APIs that meet the expectations of editing framework authors of varying skill and team sizes.   
+- allow a network of experts to share information about gaps and requirements for building editing frameworks on the web.  
+
+The specifications the Editing Taskforce is producing may be driven to the spec level directly by its members within the Web Applications Working Group, or be transferred to another, more appropriate working group after initial collection of requirements. Furthermore, the group defines its success as when it at the very least, produces a recommendation with well defined requirements that result in enhancing existing text editing related specification drafts or helps design new ones.
+
+The TF is expected to work on the following efforts until the end of the charter but it is not meant to be an exhaustive list:
+
+- [Async Clipboard API](https://bugs.chromium.org/p/chromium/issues/detail?id=931839) 
+- [ContentEditableDisabled](http://w3c.github.io/editing/contentEditableDisabled.html) - ability to disable system UI 
+- [EditContext API](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/master/EditContext/explainer.md) 
+- [Highlight API](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/master/highlight/explainer.md) 
+- Native Selection and Caret behaviors 
+- [SpellChecker API](https://github.com/w3c/editing/issues/166) 
+- [SIP Policy](https://github.com/whatwg/html/issues/4876) 
+- [Input Events](https://www.w3.org/TR/input-events-1/) 
+- [ContentEditable](https://w3c.github.io/contentEditable/) 
+
+Deliverables are expected to eventually be shipped as standards recommendation by the Web Applications Working Group, CSS Working Group or any other group where a draft specification may fall into.
+
+### 3. Participation
+
+This task force is designed to make participation relatively easy for people not currently involved in the standards process (typically, developers of editing tool libraries) who may not be amenable to signing up to a large in scope discussions that can be found in, for instance, the main Web Applications Working Group.
```

Comment:
@ylafon or @siusin This is also a bit problematic from an IPR perspective. We encourage folks to participate, but if we also want them to become members - or, at least, Invited Experts. As written, this seems at odds with the function of the W3C Working Groups (vs community groups, where we are much more liberal).

Reply:
@marcoscaceres This paragraph was part of the original taskforce charter back from when it was first set up.

Reply:
Understood, but a lot has changed since then.

Reply:
I believe the people who were previously not part of WebApps and who did join the task force did so as Invited Experts of the WG (not 100% sure this has been renewed through the various rechartering, but that was at least the intention initially). The more focused scope is relevant in terms of attention span, but certainly the IPR regime that applies has to be that of the WG as a whole.

Reply:
That is true. I and several others were made the Invited Expert of some WG, even though we had no clue about what that WG was about or working on and we would not show up to any of its meetings.

Reply:
@johanneswilm, as external folks engage with the work, to get them to become Invited Experts (the IPR bot will yell at them regardless if they try to make any pull requests, but if you see someone making substantive suggestions in bug, it's important to get them to join the WG as Invited Experts). The Chairs would be really supportive of that.

Reply:
@marcoscaceres we wanted to remove any barriers in people's participation. I agree that it is fair for people to become Invited Experts if they are ready to commit significant time and effort into editing discussions. (by the way, how do people become Invited Experts?). That said, do people need to go through the process if they just wanted to dial-in or leave comments/open issues in github?

Reply:
> by the way, how do people become Invited Experts?

It's pretty easy: the Editor's nominate them to the Chairs/Team (team-webapps@w3.org). Then they just fill out a form. 
https://www.w3.org/participate/invited-experts/

It's a quick process. 

> That said, do people need to go through the process if they just wanted to dial-in or leave comments/open issues in github?

Generally no... but it depends on what kind of contribution they are making. If they are adding something that is significant/substantive ("let's add a feature X! it would work like this: ..." or "browser behavior should follow this model: ...") then yes, they MUST become an Invited Expert - what you need to be on the lookout for is for someone claiming "this was my idea!" (because they could have a patent on it). 

If they are just fixing typos, examples, etc. then the do not need to be an Invited Expert. If unsure, ask the Chairs - and be overly cautions.

Reply:
@marcoscaceres understood, thank you. We'll update the wording in a bit.

Reply:
wording has been updated. Let us know if it works for you.